src/openxr/OpenXRApi.cpp

```diff
@@ -1161,13 +1213,21 @@ bool OpenXRApi::initialiseSwapChains() {
 			.mipCount = 1,
 		};
 
+#ifdef ANDROID
+		// Enable foveation on this swapchain
```

Comment:
Same with this, we should switch it based on the functionality being available, not on platform.

Reply:
Same here, I'll add a todo and address it in another PR.